apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -686,6 +702,25 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
           }}
         />
       )}
+      <ConfirmDialog
+        isOpen={isConfirmRenaming}
+        heading={t('Confirm renaming')}
+        confirmLabel={t('Rename')}
+        cancelLabel={t('Cancel')}
+        description={
+          (conversation.isShared &&
+            t(
+              'Renaming will stop sharing and other users will no longer see this conversation.',
+            )) ||
+          ''
+        }
+        onClose={(result) => {
+          setIsConfirmRenaming(false);
+          if (result) {
```

Comment:
on Cancel button need to disable Rename mode? Now we stay in Rename mode after Cancel

Reply:
but should set `setIsRenaming(false)` also